internal/cli/cmd/dac/setup/testdata/.gitignore

```diff
@@ -0,0 +1,3 @@
+go.sum
```

Comment:
any reason to ignore the file `go.sum` ?

Reply:
to me we don't care of tracking such file in the case of test data, but I dont mind adding it back